src/Components/Web.JS/src/Platform/Platform.ts

```diff
@@ -8,6 +8,9 @@ export interface Platform {
   toJavaScriptString(dotNetString: System_String): string;
   toUint8Array(array: System_Array<any>): Uint8Array;
 
+  toDotNetArray(array: ArrayBuffer): Pointer;
+  toDotNetString(string: String): System_String;
```

Comment:
Unless we're calling these functions from outside `MonoPlatform.ts`, we don't need to add them to the `Platform` interface. The `Platform` interface is only relevant for code that's trying not to tie itself to Mono platform internals.

TBH the `Platform` interface doesn't really strictly need to exist any more, but it's still kind of nice to have some level of decoupling from the lowest-level Mono internals in other places, so I'd vote for keeping it (just not putting more stuff in it when it's not needed by other code).